src/sentry/tagstore/snuba/backend.py

```diff
@@ -45,6 +46,13 @@
 from sentry.utils.dates import to_timestamp
 from sentry.utils.hashlib import md5_text
 
+_max_unsampled_projects = 50
+if os.environ.get("SENTRY_SINGLE_TENANT"):
```

Comment:
I know in past PRs we've tried to avoid generic "This is ST" env vars and use more functionality/feature-based naming, would using `SENTRY_ALWAYS_SAMPLE_GET_TAG_KEYS` make sense?

Reply:
Yeah, I agree.

Reply:
Hmm, in that case I would rather it be a config option. A generic environment var to me reads "this is a special exception for something that is not normally configurable". What do you think?

Reply:
As discussed, sticking with `SENTRY_SINGLE_TENANT`.